package.json

```diff
@@ -40,6 +40,11 @@
     "vite": "5.1.6",
     "vitest": "2.0.3"
   },
+  "pnpm": {
+    "overrides": {
+      "follow-redirects": "^1.15.6"
```

Comment:
This shouldn't be necessary since the semver range of the `http-proxy` package uses `^1.0.0`. We should run `pnpm update` and update the lockfile instead.

Reply:
That's what I did on https://github.com/vercel/vercel/pull/12772 and it worked but for some reason this did not update the lockfile when I tried it here. I am juggling a few branches though, so possibly it was something unrelated

Reply:
Ok, [here](https://github.com/vercel/vercel/pull/12773)'s another take. I'm sure some combination of these three will get the vuln report closed 😄

Reply:
A second attempt seems to have worked. I was probably just odd state or maybe `cd`d into a weird directory 🤷